src/main/java/com/marek/onlinebookstore/model/Status.java

```diff
@@ -0,0 +1,7 @@
+package com.marek.onlinebookstore.model;
+
+public enum Status {
+    PENDING,
+    DELIVERED,
```

Comment:
'DELIVERED' status could be more descriptive. Depending on your context, 'ORDER_DELIVERED' or 'BOOK_DELIVERED' may provide more information.